### PR TITLE
Question plugin missing assets fix

### PR DESCRIPTION
**Pull request by pallakartheekreddy**

### CODE REVIEW

File: app/scripts/contenteditor/migration/1_migration-task.js [hunk 1]
```diff
@@ -3,29 +3,58 @@
 org.ekstep.contenteditor.migration = new (Class.extend({
 	migrationFlag: false,
 	contentBackup: undefined,
+	patch: [],
 	init: function () {
 		org.ekstep.contenteditor.api.addEventListener('content:migration:start', this.execute, this)
 	},
 	_startTime: undefined,
 	tasks: ['mediamigration_task', 'basestage_task', 'orderstage_task', 'scribblemigration_task', 'imagemigration_task', 'readalongmigration_task', 'assessmentmigration_task', 'eventsmigration_task', 'settagmigration_task'],
+	questionPatchs: ['questionsetfix1'],
```
Comment (SanthoshVasabhaktula) [inline app/scripts/contenteditor/migration/1_migration-task.js]: @pallakartheekreddy - Why is this a global variable?

Comment (SanthoshVasabhaktula) [inline app/scripts/contenteditor/migration/1_migration-task.js]: @pallakartheekreddy - Can we rename "questionsetfix1" as "patch_1" or may be 6 characters alphanumeric value?

File: app/scripts/contenteditor/migration/1_migration-task.js [hunk 2]
```diff
@@ -82,5 +111,32 @@ org.ekstep.contenteditor.migration = new (Class.extend({
 	},
 	getBackupContent: function () {
 		return this.contentBackup
+	},
+	isAssessmentContent: function (contentbody) {
+		var instance = this
+		var assessmentContent = false
+		if (!_.isEmpty(instance.patch)) {
```
Comment (SanthoshVasabhaktula) [inline app/scripts/contenteditor/migration/1_migration-task.js]: @pallakartheekreddy - This check is redundant

File: app/scripts/contenteditor/migration/1_migration-task.js (hunk 1 above)
Comment (pallakartheekreddy) [reply app/scripts/contenteditor/migration/1_migration-task.js]: Yes, we can use `patch_1`.  list of patches I am maintaining an array to verify and run the migration patch.

File: app/scripts/contenteditor/migration/1_migration-task.js (hunk 2 above)
Comment (pallakartheekreddy) [reply app/scripts/contenteditor/migration/1_migration-task.js]: Yes, this is not required I'll remove.

**Pull request merged**